README.md

```diff
@@ -1,35 +1,35 @@
-
-> The [Fishshell][fishshell] Framework
-
-[![Fish Version][fish-badge]][fishshell]
-[![Build Status][travis-badge]][travis-url]
-[![License][license-badge]](#LICENSE)
-
-<a name="omf"></a>
-<br>
-
 <p align="center">
   <a href="https://github.com/oh-my-fish/oh-my-fish/blob/master/README.md">
-  <img width="160px" src="https://cloud.githubusercontent.com/assets/8317250/8510172/f006f0a4-230f-11e5-98b6-5c2e3c87088f.png">
+    <img width="160px" src="https://cloud.githubusercontent.com/assets/8317250/8510172/f006f0a4-230f-11e5-98b6-5c2e3c87088f.png"/>
   </a>
 </p>
 
-<br>
+<p align="center">
+  The <a href="http://fishshell.com">Fishshell</a> Framework
+</p>
 
 <p align="center">
-<b><a href="#about">About</a></b>
-|
-<b><a href="#install">Install</a></b>
-|
-<b><a href="#getting-started">Getting Started</a></b>
-|
-<b><a href="#advanced">Advanced</a></b>
-|
-<b><a href="https://github.com/oh-my-fish/oh-my-fish/wiki/Screencasts">Screencasts</a></b>
-|
-<b><a href="/CONTRIBUTING.md">Contributing</a></b>
-|
-<b><a href="/FAQ.md">FAQ</a></b>
+  <img src="https://img.shields.io/badge/license-MIT-007EC7.svg?style=flat-square" alt="MIT license"/>
+  <img src="https://img.shields.io/badge/fish-v2.2.0-007EC7.svg?style=flat-square" alt="Fish version"/>
+  <a href="https://travis-ci.org/oh-my-fish/oh-my-fish">
+    <img src="http://img.shields.io/travis/oh-my-fish/oh-my-fish.svg?style=flat-square" alt="Travis build status"/>
+  </a>
+</p>
+
+<p align="center">
+  <b><a href="#about">About</a></b>
```

Comment:
These references are not working. Could you fix them? They are already broken on master.